### PR TITLE
Release 3.2.0

### PR DESCRIPTION
**Pull request by hardbyte**
Promotes the beta release to final and merges into master.

Try the beta with:
```
pip install python-can==3.2.0b0
```

Changelog notes are on the beta release page - https://github.com/hardbyte/python-can/releases/tag/3.2.0-beta.0

All the issues and PRs for this release are closed in the [milestone](https://github.com/hardbyte/python-can/milestone/7?closed=1).

### CODE REVIEW

File: CHANGELOG.txt [hunk 1]
```diff
@@ -1,3 +1,62 @@
+Version 3.2.0
+====
+
```
Comment (felixdivo) [inline CHANGELOG.txt]: Maybe we should note that this version is the last to support Python 2.7 (and Python 3.5). We could note that here in the changelog and in the Readme? Maybe we should add a small table in the docs that we link to in the Readme, like:

Our version | Supported Python versions
4.0 | 3.6+ (expected)
3.2 | 2.7, 3.5+
3.1 | 2.7, 3.4+
2.0 | 2.7, 3.2+

(I did not actually look up the versions, just as an example)

Comment (hardbyte) [reply CHANGELOG.txt]: Good idea - done.

Comment (felixdivo) [reply CHANGELOG.txt]: Nice! :)

File: CHANGELOG.txt [hunk 2]
```diff
@@ -1,3 +1,64 @@
...
+
+Major features
+--------------
+
+* FD support added for Pcan by @bmeisels with input from
+  @markupspi, @christiansandberg & @felixdivo in PR #537
```
Comment (karlding) [inline CHANGELOG.txt]:
```
s/markupspi/markuspi
```

Comment (hardbyte) [reply CHANGELOG.txt]: 👀 good catch

**Comment by hardbyte** [comment]
I've opened issues #584 #585 #586 to address the fact the diff isn't tested to the level we've been aiming for.

**Pull request merged**